packages/babel-plugin-transform-classes/test/fixtures/assumption-constantSuper/accessing-super-class/output.js

```diff
@@ -2,16 +2,15 @@ var Test = /*#__PURE__*/function (_Foo) {
   "use strict";
 
   babelHelpers.inherits(Test, _Foo);
-  var _super = babelHelpers.createSuper(Test);
   function Test() {
     var _Foo$prototype$test;
     var _this;
     babelHelpers.classCallCheck(this, Test);
     woops.super.test();
-    _this = _super.call(this);
+    _this = babelHelpers.callSuper(this, Test);
     _Foo.prototype.test.call(babelHelpers.assertThisInitialized(_this));
-    _this = _super.apply(this, arguments);
-    _this = _super.call.apply(_super, [this, "test"].concat(Array.prototype.slice.call(arguments)));
+    _this = babelHelpers.callSuper(this, Test, arguments);
+    _this = babelHelpers.callSuper(this, Test, ["test"].concat(Array.prototype.slice.call(arguments)));
```

Comment:
This seems to be a bug fix, `this` should not be taken as a parameter.

There's a similar old behavior below, but to keep this PR small enough I'll deal with it in the future.